src/components/SecondaryPanes/Expressions.css

```diff
@@ -72,6 +72,7 @@
   background-color: var(--theme-body-background);
   display: block;
   position: relative;
+  min-height: 2.8em;
```

Comment:
I think we'll need to do a bit more than this; specifically, I think we should set the same height for both expression items and breakpoint items, so we'll need the height chosen to be in both places.